gateway/1.8/modules/ROOT/pages/flex-sizing-guide.adoc

```diff
@@ -1,4 +1,4 @@
-= Reviewing Resource Sizing 
+= Resource Sizing for Flex Gateway
```

Comment:
This topic seems to be a group of tasks (based on the imperative H2s), but the title is more of a concept/reference.
Perhaps this is a "Planning" task? Either that, or you could change the H2 tasks to be referencey/concepty.

Reply:
I changed the H2 tasks to be more referencey, but this topic needs a rewrite. I don't have enough time or information to do that right now, so I created a backlog item for a rewrite.